aea/manager/manager.py

```diff
@@ -263,8 +267,11 @@ async def _manager_loop(self) -> None:
                 agent_name = agents_run_tasks_futures[task]
                 self._agents_tasks.pop(agent_name)
                 if task.exception():
-                    for callback in self._error_callbacks:
-                        callback(agent_name, task.exception())
+                    if len(self._error_callbacks) == 0:
+                        self._print_exception_occurred_but_no_error_callback(agent_name)
```

Comment:
make it default callback?

Reply:
Yes that's a good idea. Why don't we have a default callback?

Reply:
if someone adds a callback with `add_error_callback`, should the default callback be replaced?

Reply:
Yes